tutorials/Mouse_Events/README.md

```diff
@@ -221,26 +289,68 @@ import androidx.compose.runtime.mutableStateOf
 import androidx.compose.runtime.remember
 import androidx.compose.runtime.setValue
 import androidx.compose.ui.Alignment
+import androidx.compose.ui.ExperimentalComposeUiApi
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.input.pointer.*
+import androidx.compose.ui.awt.awtEvent
+import androidx.compose.ui.input.pointer.PointerEventType
+import androidx.compose.ui.input.pointer.onPointerEvent
 import androidx.compose.ui.window.singleWindowApplication
 
+@OptIn(ExperimentalComposeUiApi::class)
 fun main() = singleWindowApplication {
     var text by remember { mutableStateOf("") }
 
     Box(
-        Modifier.fillMaxSize().pointerInput(Unit) {
-            while (true) {
-                val event = awaitPointerEventScope { awaitPointerEvent() }
-                val awtEvent = event.mouseEvent
-                if (event.type == PointerEventType.Press) {
-                    text = awtEvent?.locationOnScreen?.toString().orEmpty()
-                }
-            }
-        },
+        Modifier
+            .fillMaxSize()
+            .onPointerEvent(PointerEventType.Press) {
+                text = it.awtEvent.locationOnScreen?.toString().orEmpty()
+            },
         contentAlignment = Alignment.Center
     ) {
         Text(text)
     }
 }
 ```
+*Note that onPointerEvent is experimental and can be changed in the future. For more stable API look at [Modifier.pointerInput](#listenining-raw-events-in-commonmain-via-modifierpointerinput)*.
+
+### Listenining raw events in commonMain via Modifier.pointerInput
+Multiplatform source set commonMain doesn't support experimental `Modifier.onPointerInput` yet. Instead of it you can use stable Modifier.pointerInput:
```

Comment:
We probably need to explain why people would want onPointerEvent at all.

Reply:
Here you likely meant `onPointerEvent` in first sentence.

Reply:
Done